### PR TITLE
Initialized project

### PR DESCRIPTION
**Pull request by apereverzin**
Updated after creating the repository from spring-boot-template, defined charts and Jenkins files

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[x] No
```

### CODE REVIEW

**Comment by rkondratowicz** [comment]
The gateway stuff can be removed afaik

File: Jenkinsfile_nightly [hunk 1]
```diff
@@ -9,7 +9,7 @@ properties([
 
 def type = "java"
 def product = "rpe"
```
Comment (doncem) [inline Jenkinsfile_nightly]:
```suggestion
def product = "reform-scan"
```

File: Jenkinsfile_nightly [hunk 2]
```diff
@@ -9,7 +9,7 @@ properties([
...
-def component = "spring-boot-template"
+def component = "reform-scan-notification-service"
```
Comment (doncem) [inline Jenkinsfile_nightly]:
```suggestion
def component = "notification-service"
```

File: Jenkinsfile_nightly [hunk 3]
```diff
@@ -9,7 +9,7 @@ properties([
...
 
 withNightlyPipeline(type, product, component) {
   enableSlackNotifications('#rpe-build-notices')
```
Comment (doncem) [inline Jenkinsfile_nightly]:
```suggestion
  enableSlackNotifications('#bsp-build-notices')
```

File: README.md [hunk 4]
```diff
@@ -1,102 +1,9 @@
...
-
-Located in `./bin/init.sh`. Simply run and follow the explanation how to execute it.
-
-## Notes
-
-Since Spring Boot 2.1 bean overriding is disabled. If you want to enable it you will need to set `spring.main.allow-bean-definition-overriding` to `true`.
-
-JUnit 5 is now enabled by default in the project. Please refrain from using JUnit4 and use the next generation
+![](https://github.com/hmcts/reform-scan-notification-service/workflows/CI/badge.svg)
+[![](https://github.com/hmcts/reform-scan-notification-service/workflows/Publish%20Swagger%20Specs/badge.svg)](https://hmcts.github.io/reform-api-docs/swagger.html?url=https://hmcts.github.io/reform-api-docs/specs/reform-scan-notification-service.json)
```
Comment (doncem) [inline README.md]: these workflows don't exist

File: README.md [hunk 5]
```diff
@@ -1,102 +1,9 @@
...
+[![codecov](https://codecov.io/gh/hmcts/reform-scan-notification-service/branch/master/graph/badge.svg)](https://codecov.io/gh/hmcts/reform-scan-notification-service)
+[![Codacy Badge](https://api.codacy.com/project/badge/Grade/c99af8bcd53947deb32e8f0a7c500676)](https://www.codacy.com/manual/HMCTS/reform-scan-notification-service)
```
Comment (doncem) [inline README.md]: those 2 don't exist either

File: src/main/resources/application.yaml [hunk 6]
```diff
@@ -8,31 +8,7 @@ management:
   endpoints:
     web:
       base-path: /
-      exposure:
-        include: health, info, prometheus
```
Comment (doncem) [inline src/main/resources/application.yaml]: i think those are needed for pipeline interactions (in future). except `prometheus`

File: Jenkinsfile_CNP [hunk 7]
```diff
@@ -0,0 +1,70 @@
...
+}
+
+GradleBuilder builder = new GradleBuilder(this, product)
+
+withPipeline(type, product, component) {
+  after('test') {
+    builder.gradle('integration')
+  }
+
+  enableDbMigration('reform-scan') // vault 'prefix'
```
Comment (doncem) [inline Jenkinsfile_CNP]: not needed for now

File: Dockerfile [hunk 8]
```diff
@@ -5,7 +5,7 @@ ARG APP_INSIGHTS_AGENT_VERSION=2.5.1
 FROM hmctspublic.azurecr.io/base/java:openjdk-11-distroless-1.2
```
Comment (doncem) [inline Dockerfile]: we should use v1.4 so default encoding is passed to jvm properly

File: Jenkinsfile_CNP (hunk 7 above)
Comment (apereverzin) [reply Jenkinsfile_CNP]: done

File: Dockerfile (hunk 8 above)
Comment (apereverzin) [reply Dockerfile]: done

File: src/test/java/uk/gov/hmcts/reform/notificationservice/ApplicationTest.java [hunk 9]
```diff
@@ -0,0 +1,13 @@
+package uk.gov.hmcts.reform.notificationservice;
+
+import org.junit.jupiter.api.Test;
+
+import static org.assertj.core.api.Assertions.assertThat;
+
+class ApplicationTest {
+    @Test
+    void dummyTest() {
```
Comment (nitinprabhu) [inline src/test/java/uk/gov/hmcts/reform/notificationservice/ApplicationTest.java]: Why can't we have health endpoint test in integration ?

Comment (apereverzin) [reply src/test/java/uk/gov/hmcts/reform/notificationservice/ApplicationTest.java]: It is a unit test - just to pass checks

Comment (nitinprabhu) [reply src/test/java/uk/gov/hmcts/reform/notificationservice/ApplicationTest.java]: You can write an integration tests to verify health.

**Pull request merged**